### PR TITLE
Changed Person equals logic. Change find command to search if a name contains a word instead of matching perfectly to the name.

### PR DESCRIPTION
**Pull request by igezt**
Closes issue #108 and #113

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -127,6 +127,31 @@ public Address getAddress() {
         return address;
     }
```
Comment (igezt) [inline src/main/java/seedu/address/model/person/Person.java:129]: I moved the following methods up in the Person class to group them with the other getter methods.

File: src/main/java/seedu/address/model/person/fields/enums/Genders.java [hunk 2]
```diff
@@ -9,10 +9,15 @@ public enum Genders {
     FEMALE("female"),
     DNS("dns");
 
-    private final String gender;
+    public final String gender;
```
Comment (hansstanley) [inline src/main/java/seedu/address/model/person/fields/enums/Genders.java:12]: Think this can be private from your subsequent usage.

**Review by hansstanley — APPROVED** [review]
Minor changes but LGTM!

File: src/main/java/seedu/address/model/person/fields/enums/Genders.java (hunk 2 above)
Comment (igezt) [reply src/main/java/seedu/address/model/person/fields/enums/Genders.java]: Gotcha. Change added

**Pull request merged**